src/crimson/mgr/client.cc

```diff
@@ -100,29 +101,28 @@ seastar::future<> Client::handle_mgr_conf(crimson::net::Connection* conn,
                                           Ref<MMgrConfigure> m)
 {
   logger().info("{} {}", __func__, *m);
-  tick_period = std::chrono::seconds{m->stats_period};
-  if (tick_period.count() && !tick_timer.armed() ) {
-    tick();
+
+  auto tick_period = std::chrono::seconds{m->stats_period};
+  if (tick_period.count()) {
+    if (tick_timer.armed()) {
+      tick_timer.rearm(tick_timer.get_timeout(), tick_period);
+    } else {
+      tick_timer.arm_periodic(tick_period);
```

Comment:
Just looked at seastar::timer, it seems that `tick_timer.arm_periodic(...)` won't call `tick()` immediately when set up, which is different from the previous behavior.
Maybe change to
```
tick_timer.arm(seastar::lowres_clock::now(), tick_period);
```

Reply:
@cyx1231st i am aware of this behavior. but i'd prefer avoid repeating `seastar::lowres_clock` if possible, since mgr client updates mgr with stats and non-critical information, i think it's fine to delay a little bit for the first update.